.github/workflows/mainnet_deb_profiles.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout
-        uses: actions/checkout@v4
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
```

Comment:
same here and for action-gh-release
Any reason why we're using a commit hash instead of a stable release?

Reply:
https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions